src/CashuWallet.ts

```diff
@@ -40,16 +40,17 @@ import { wordlist } from '@scure/bip39/wordlists/english';
 class CashuWallet {
 	private _keys: MintKeys | undefined;
 	private _seed: Uint8Array | undefined;
+  private _unit = 'sat';
 	mint: CashuMint;
-	unit = 'sat';
+	
 
 	/**
 	 * @param keys public keys from the mint
 	 * @param mint Cashu mint instance is used to make api calls
 	 * @param mnemonicOrSeed mnemonic phrase or Seed to initial derivation key for this wallets deterministic secrets. When the mnemonic is provided, the seed will be derived from it.
 	 * This can lead to poor performance, in which case the seed should be directly provided
 	 */
-	constructor(mint: CashuMint, keys?: MintKeys, mnemonicOrSeed?: string | Uint8Array) {
+	constructor(mint: CashuMint, unit?:string, keys?: MintKeys, mnemonicOrSeed?: string | Uint8Array) {
```

Comment:
we pass in `unit` here, but do nothing with it? Why?

Reply:
working on it. I think the merge didn't port the changes. Will test and push it.

Reply:
ok. fixed it. To test try this. 

//For creating invoice using Strike backend
const wallet = new CashuWallet(new CashuMint('https://stablenut.umint.cash'), 'usd');

//For creating invoice using LND backend
const wallet = new CashuWallet(new CashuMint('https://stablenut.umint.cash'));